week-4-practise/level-2/exercise-1.js

```diff
@@ -11,13 +11,13 @@ We made a start on this for you here:
 var bottle = {
   volume: 0,
   fill: function() {
-    // calling this function should make you bottles volume = 100;
+    return (this.volume = this.volume + 100); // calling this function should make you bottles volume = 100;
   },
   drink: function() {
-    // calling this function should decrease your bottles volume by 10;
+    return (this.fill = this.fill - 10); // calling this function should decrease your bottles volume by 10;

```

Comment:
Should use `this.volume` here, not `this.fill`. Again nothing needs to be returned.